ArchiSteamFarm/ArchiWebHandler.cs

```diff
@@ -142,7 +142,7 @@ public sealed class ArchiWebHandler : IDisposable {
 				await ASF.InventorySemaphore.WaitAsync().ConfigureAwait(false);
 
 				try {
-					WebBrowser.ObjectResponse<Steam.InventoryResponse>? response = await UrlGetToJsonObjectWithSession<Steam.InventoryResponse>(SteamCommunityURL, request + (startAssetID > 0 ? "&start_assetid=" + startAssetID : "")).ConfigureAwait(false);
+					WebBrowser.ObjectResponse<Steam.InventoryResponse>? response = await UrlGetToJsonObjectWithSession<Steam.InventoryResponse>(SteamCommunityURL, request + (startAssetID > 0 ? "&start_assetid=" + startAssetID : ""), requestOptions: WebBrowser.ERequestOptions.ReturnServerErrors).ConfigureAwait(false);
```

Comment:
And what is the reasoning of not retrying to access the inventory that, has returned server error code? Thanks to this logic, the retrieval of inventory is reliable despite of occassional Steam interval errors that happen between one request and another, so I hope you have a good explanation for doing that.

Reply:
```
{"error":"EYldRefreshAppIfNecessary failed with EResult 20"}
```
sometimes i got error like that in csgo and dota2

Reply:
EResult of 20 is `ServiceUnavailable` and is one of the few things that scream `FUC***G RETRY ME` like nothing else.

Reply:
And? Steam outage that lasted for 18 minutes is your reasoning for not retrying in every scenario, where the error code clearly says that it's a temporary issue? I have enough of examples where retrying on `5xx` was successful, your point?